dist/dialog.linker.build.js

```diff
@@ -150,26 +170,24 @@ window.jQuery = window.jQuery || window.shoestring;
 			var link = $a.is( "[data-dialog-link]" );
 			var iframe = $a.is( "[data-dialog-iframe]" );
 
-
 			function createDialog(content){
-				var linkId = $a.attr( "id" );
+				var linkHref = $a.attr( "href" );
 				var dialogClasses = $a.attr( "data-dialog-addclass" ) || "";
+				var dialogNoHistory = $a.is( "[data-dialog-nohistory]" );
 				var id;
 
-				if( linkId ) {
-					id = linkId + "-dialog";
-				} else {
-					id = "dialog-" + new Date().getTime();
+				if( linkHref ) {
+					id = linkHref;
 				}
 
 				$a
 					.attr("href", "#" + id )
```

Comment:
It seems like if we're assuming the `id` value comes from the link's `href` attribute we don't need to set it again with an extra hash right?